project_task_material_stock/tests/test_task_material.py

```diff
@@ -1,111 +1,119 @@
 # Copyright 2015 Tecnativa - Sergio Teruel
 # Copyright 2015 Tecnativa - Carlos Dauden
 # Copyright 2016 Tecnativa - Vicent Cubells
+# Copyright 2020 jeo Software Jorge Obiols <jorge.obiols@gmail.com>
```

Comment:
I'm not sure it's well received to insert copyrights claims when you just reformat a file like that...